### ISSUE TITLE
Available GB is not what is expected

### ISSUE THREAD

**Issue by TacoByte** [NebulousLabs/Sia-UI#330]
I bought 10 GB. I've been playing around uploading a few small files.

The UI is reporting that I have 2GB left!

**582 KB Used / 2 GB Available**

```
Renter info:
    Storage Spending:  606.6 mS
    Upload Spending:   21.02 SC
    Download Spending: 0 H
    Unspent Funds:     4.146 KS
    Total Allocated:   4.167 KS

Tracking 6 files:
 252.0 KB  1453756944023.png
   6.1 KB  untitled folder 3/.DS_Store
   7.8 KB  untitled folder 3/1453756843707.jpg
 252.0 KB  untitled folder 3/1453756944023.png (uploading, 8.33%)
   8.0 KB  untitled folder 3/untitled folder/1443814339804.jpg (uploading, 8.33%)
  56.4 KB  untitled folder 3/untitled folder/1452160563683.jpg (uploading, 8.33%)
```

**Comment by avahowell** [comment]
What is the output of `siac renter allowance`?

**Comment by TacoByte** [comment]
```
Allowance:
    Amount: 1.548 KS
    Period: 12960 blocks
```

**Comment by avahowell** [comment]
Okay, so I've verified that the storage estimation calculation is isomorphic (there's no error in the calculation of how much "storage" you have with a given allowance.)  The reason this happens is that there can be price volatility in the host db.  The average price may be lower or higher when you "buy storage" than when you use the Files plugin.  "Buying storage" is really a leaky abstraction, what you're actually doing is setting a maximum allowance for the UI to spend on storage on your behalf.  This allowance is in coins and can correspond to more or less storage depending on the average hostdb price.

It may be clearer to simply show disk usage and spending instead of trying to calculate storage capacity from the allowance.

**Comment by TacoByte** [comment]
Gotcha. Yeah, I guess this is one more problem that is made worse because of a tiny host ecosystem and high SC price volatility. It might be a good idea to change the whole "buy" idea into something else like what you say. My expectation was definitely that I would effectively buy/reserve/pre-pay a given amount of GB and that I would use it to my liking (and then get refunded what I didn't use).

**Comment by TacoByte** [comment]
I'm now getting `582 KB Used / 2 MB Available`!

**Comment by avahowell** [comment]
fixed in #409
